### ISSUE TITLE
No audio Big Sur 

### ISSUE THREAD

**Issue by ike1985** [themagiulio/bandcamp-desktop#18]
I'm on a new macbook pro with Big Sur and I get no audio at all when I try to play anything from the desktop app.

**Comment by Fospunk** [comment]
I'm having the same issue.

**Comment by themagiulio** [comment]
Hi, thanks for reporting the issue.
Unfortunately I don't have a mac with Big Sur, so I'm not able to find and solve the problem.
I can suggest you to compile the software on your own by following [these instructions](https://github.com/themagiulio/bandcamp-desktop/issues/13#issuecomment-764798433).

If you succeed to do it and it works, I suggest you to publish here the compiled dmg file, so if there are other people in the same situation they will just need to download the installer.

**Comment by montemartin** [comment]
I just downloaded, and I'm having the same issue, but it's only happening when I play anything from my profile page. If I play music from an album page or elsewhere it works fine. I'm also on an M1 Mac so that could be contributing. 

Before I file a separate issue I'm curious if others are experiencing it only on the profile page?

**Comment by orangecoloured** [comment]
> Before I file a separate issue I'm curious if others are experiencing it only on the profile page?

Correct. Album page works fine. Profile page music doesn't have audio.
